### ISSUE TITLE
 Filter functionality in Always show object details

### ISSUE THREAD

**Issue by Zero2Success** [cvat-ai/cvat#8167]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://docs.cvat.ai/docs/)

### Is your feature request related to a problem? Please describe.

Hi I'm a labeler using Cvat!
First of all, thank you guys for making Cvat open source!
I'm labeling words professionally.

I've been using your cvat and found a feature called Always show object details, so I tried it out!
It's a really useful feature,
but it's hard to see when there are a lot of attributes.
So I wish Attributes could be a filter and Always show object details would work, so it would be easier to see.

Here's my crude code for it.

**cvat/cvat-canvas/src/typescript/canvasView.ts**

`if (withAttr) {
                    const { displayAllText } = this.configuration;
                    for (const attrID of Object.keys(attributes)) {
                        const attrName = attrNames[attrID];
                        const value = attributes[attrID] === undefinedAttrValue ? '' : attributes[attrID];
                        if (displayAllText) {
                            if (attrName === 'text') {
                                block
                                    .tspan(`${value}`)
                                    .attr({
                                        attrID,
                                        dy: '1em',
                                        x: 0,
                                    })
                                    .addClass('cvat_canvas_text_attribute');
                            }
                        }else {
                            block
                                .tspan(`${attrName}: ${value}`)
                                .attr({
                                    attrID,
                                    dy: '1em',
                                    x: 0,
                                })
                                .addClass('cvat_canvas_text_attribute');
                        }
                    }
                }`
This made me more productive because I could see all my Word edits at once!

![image](https://github.com/user-attachments/assets/3ce4aa46-fc9d-4b8b-9cf1-5579f45168f1)

This is the image applied using the code.

Bottom line:
I wish there was a filter function for always show object details!

P.S. I'm Korean, so I apologize in advance for my broken English.

### Describe the solution you'd like

I wish there was a filter function for always show object details!
